PerfectNumberTest.java

```diff
@@ -11,4 +11,10 @@ public void should_return_true_when_6_is_passed() {
         assertTrue(perfectNumber.isPerfectNumber(6));
     }
 
+    @Test
+    public void should_return_true_As_1_is_factorOf_6() {
+        PerfectNumber perfectNumber = new PerfectNumber();
+        assertTrue(perfectNumber.isFactor(1,6));
+    }
+

```

Comment:
You cant simply write another method. It should evolve during the refactor step. I didnt see any corresponding test cases which would have made you write this method.